docs/dev/install.rst

```diff
@@ -194,3 +194,69 @@ If you ever need to bypass Therapist, you can do so by passing
 ``--no-verify`` to your ``git commit`` command.
 
 .. _Therapist: http://therapist.readthedocs.io/en/latest/overview.html
+
+.. _remotesettings-install:
+
+Remote Settings
+---------------
+If you want to enable the publication of recipes on `Remote Settings`_, you'll need to set up
+an instance locally.
+
+1. Follow the `Remote Settings installation instructions`_ to launch a development
+   instance of Remote Settings locally.
+
+2. Add the following configuration to ``.env`` (create the file
+   if it does not exist yet):
+
+   .. code-block:: ini
+
+      DJANGO_REMOTE_SETTINGS_URL=http://localhost:8888/v1
+      DJANGO_REMOTE_SETTINGS_USERNAME=normandy
+      DJANGO_REMOTE_SETTINGS_PASSWORD=n0rm4ndy
+
+3. Add the following configuration to ``server.ini`` and restart the Remote Settings container:
```

Comment:
It isn't clear that `server.ini` and `.env` belong in different projects. Can you mention where these files should go?